modules/channel-web/src/views/lite/components/messages/MessageGroup.tsx

```diff
@@ -88,17 +88,18 @@ class MessageGroup extends React.Component<Props> {
             <span data-from={fromLabel} className="from hidden" aria-hidden="true">
               {fromLabel}
             </span>
-            {sortBy(messages, ['sent_on', 'eventId']).map((message, i, messages) => {
-              const isLastMsg = i === messages.length - 1
+            {sortBy(messages, ['sent_on', 'eventId']).map((message, i) => {
               let payload = this.convertPayloadFromOldFormat(message)
+
               if (payload?.wrapped) {
```

Comment:
I don't think payload can be undefined here.

```suggestion
              if (payload.wrapped) {
```